tox.ini

```diff
@@ -43,6 +43,9 @@ deps =
     dj31: Django>=3.1,<3.2
     dj32: Django>=3.2,<3.3
     dj40: Django>=4.0,<4.1
+    dj42: Django>=4.2,<4.3
+    dj50: Django>=5.0,<5.1
+    dj51: Django>=5.1,<5.2
     djmain: https://github.com/django/django/archive/main.tar.gz
```

Comment:
if main was removed from the testing above, this too can be removed. it was used to test the changes with the bleeding edge of django, to allow for faster feedback.